⚠️ CONFLICT! Lineage pull request for: skeleton (#106)
cisagov/vdp-scanner-docker: README.md

1. Pull the new image:

    ```console
    docker pull cisagov/vdp-scanner:0.2.0-dev.5
    ```

1. Recreate and run the container by following the [previous instructions](#running-with-docker).

## Updating Python dependencies ##

This image uses [Pipenv] to manage Python dependencies using a [Pipfile](https://github.com/pypa/pipfile).
Both updating dependencies and changing the [Pipenv] configuration in `src/Pipfile`
will result in a modified `src/Pipfile.lock` file that should be committed to the
repository.

> [!WARNING]
> The `src/Pipfile.lock` as generated will fail `pre-commit` checks due to JSON formatting.

### Updating dependencies ###

If you want to update existing dependencies you would run the following command
in the `src/` subdirectory:

```console
pipenv lock
```

### Modifying dependencies ###

If you want to add or remove dependencies you would update the `src/Pipfile` file
and then update dependencies as you would above.

> [!NOTE]

> > You should only specify packages that are explicitly needed for your Docker

Reviewer: If I understand what you're saying, then I think this wording is clearer:
```suggestion
> You should only specify packages that are direct requirements of your Docker
```

Reviewer 2: This is from upstream so the change should be made there.

Reviewer: See cisagov/skeleton-docker#209.
